components/applications-service/integration_test/service_groups_sorting_test.go

```diff
@@ -218,11 +218,13 @@ func TestGetServiceGroupsSortedPercent(t *testing.T) {
 	response, err := suite.ApplicationsServer.GetServiceGroups(ctx, request)
 	assert.Nil(t, err)
 
-	if assert.Equal(t, 4, len(response.ServiceGroups)) {
+	if assert.Equal(t, 6, len(response.ServiceGroups)) {
```

Comment:
Sorting behavior shouldn't be changed but these test were relying on the bug behavior so they had to be changed.